editor/javascript/EditorApplication.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
Not that it matters, but I think you could just do `import React, { Component } from "react";`